### ISSUE TITLE
What should autoroll do if you're already on jackpot?

### ISSUE THREAD

**Issue by TerryCavanagh** [TerryCavanagh/diceydungeons.com#396]
Right now, it does nothing, wasting the limit break. What else might be interesting?

 - It could reset the counter, essentially allowing you to get a lot of extra dice and a second chance at jackpot (which isn't as good as it sounds, unless you've got something reusable)
 - It could give you an additional jackpot reward, like "fixed payout" does. This is somewhat logical, because the idea is that it guarantees a jackpot.
 - It could do something totally different.

Think about this for v0.11!

**Comment by YinYin-Falcon** [comment]
The logical thing. Second jackpot action.

Also bring up another jackpot card if you've already gotten _and_ used a jackpot this round.
(the fact that it neither flies out to come back nor uses the multi card stack on fixed payout is a bit irritating)

Alternatively, just bump up the jackpot actions. (10 damage, 6 heal, two dice)
This makes it a bit less versatile alone, but a more interesting combination with fixed payout (the effects multiply rather than add up).

I wouldn't go for entirely different rewards, because that'd be a bit unexpected and much harder to remember/plan for.

The counter reset would only make sense to me, if you completely reframe the limit to be a stack purge in any case.

**Comment by onetruepurple** [comment]
I don't think it should do anything - we don't have a "What should Snap do if you don't have any double cards in hand" issue for the Jester.

**Comment by YinYin-Falcon** [comment]
If a limit can be disabled for not being applicable, sure why not. Might feel a little disappointing though. That usually happens when you get silenced.
